usautobuild/actions/gitter.py

```diff
@@ -57,3 +57,31 @@ def update_repo(self) -> None:
     def start_gitting(self) -> None:
         self.prepare_git_directory()
         self.config.project_path = self.local_repo_dir / "UnityProject"
+
+    def get_Good_file_tag(self) -> str:
```

Comment:
move this to your good files class

Reply:
*would this involve copying the functionality of gitter tho?